src/components/action/validation/ValidatinAction.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Action from '../Action';
+import ValidationSettings from './validationActionComponents/ValidationSettings';
+
+class ValidationAction extends Component {
+    render() {
+        return (
+            <Action
+                checked={this.props.checked}
+                title="Validation"
+                collapsible
+                onToggleActive={this.props.onToggleActive}
+            >
+                <ValidationSettings {...this.props} />
```

Comment:
I have doubts about passing props this way... I think it would be cleaner if ValidationAction.container's mapStateToProps fully corresponded to ValidationAction propTypes, easier to notice changes and obsolete props in the future. 
So, let's probably wrap ValidationSettings in its own container and leave ValidationAction.container simple.